modules/monitoring-setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects.adoc

```diff
@@ -3,10 +3,16 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="setting-scrape-sample-and-label-limits-for-user-defined-projects_{context}"]
-= Setting scrape sample and label limits for user-defined projects
+[id="setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects_{context}"]
+= Setting scrape and evaluation intervals and enforced limits for user-defined projects
```

Comment:
```suggestion
= Setting scrape and evaluation intervals for enforced limits in user-defined projects
```

Reply:
The double `and` reads awkward. Maybe: `Setting the scrape and evaluation intervals as well as enforced limits for user-defined projects`.

Reply:
Hello, I think the first suggestion would change the meaning, we have scrape and evaluation intervals and then we have enforced limits. it is not for enforced limits.  Not sure if "as well as" sounds better :thinking: 

I am thinking, this could work the best: `Setting scrape intervals, evaluation intervals, and enforced limits..`

Thank you for pointing this out!